.travis.yml

```diff
@@ -112,6 +112,9 @@ matrix:
   allow_failures:
     - php: nightly
 
+after_script:
+  - sh .travis.coverage.sh
```

Comment:
I'd rather keep the config here so we won't have to maintain multiple files (kind of what we have in the ORM)